src/administration/web/configure-project.md

```diff
@@ -37,10 +34,37 @@ If you check the `Super user` box, this user will be an administrator of the pro
 
 The `Domains` screen allows you to manage your domains that your project will be accessible at.
 
-![Manage domains of your Platform.sh project.](/images/ui-conf-project-domains.png)
+![project domain](/images/settings_domain.png)
 
 More information on how to [setup your domain](/golive/steps.md).
 
 > **note**
 >
 > Platform.sh expects an ASCII representation of the domain here. In case you want to use an internationalized domain name you can use the [conversion tool provided by Verisign](http://mct.verisign-grs.com/) to convert your IDN domain to ASCII.
+
+
+## Certificates
+
+The `Certificates` screen allows you to manage your project's TLS certificates that enable HTTPS. 
+
+![project certs](/images/settings_certificates.png)
+
+You can view current certificates by hovering over one on the list and clicking the `View` link that appears, or you can add a new certificate by clicking the `Add` button a the top of the page. 
+
+![add certs](/images/add_cert.png)
```

Comment:
We should add here that because of Let's Encrypt you rarely need to touch this.  Proposal:

```
All projects get TLS certificates provided by Let's Encrypt automatically.  In most cases no user action is required.  You will only need to add certificates on this page if you are using TLS certificates provided by a third party.
```